publication-commons/src/main/java/no/unit/nva/publication/model/business/GeneralSupportRequest.java

```diff
@@ -62,7 +63,7 @@ public static GeneralSupportRequest create(Resource resource, UserInstance userI
         generalSupportRequest.setModifiedDate(Instant.now());
         generalSupportRequest.setStatus(TicketStatus.PENDING);
         generalSupportRequest.setIdentifier(SortableIdentifier.next());
-        generalSupportRequest.setViewedBy(ViewedBy.addAll(userInstance.getUser()));
+        generalSupportRequest.setViewedBy(Set.of());
```

Comment:
Here as elsewhere, Collections::emptySet

Reply:
Will do it. We run following for every ticket on creation:
```
   protected TicketEntry() {
        viewedBy = ViewedBy.empty();
    }
```